packages/block-library/src/image/edit.native.js

```diff
@@ -286,13 +277,6 @@ class ImageEdit extends React.Component {
 					{ ( ! this.state.isCaptionSelected ) &&
 						getToolbarEditButton( openMediaOptions )
 					}
-					<InspectorControls>
-						<ToolbarButton
-							title={ __( 'Image Settings' ) }
-							icon="admin-generic"
-							onClick={ onImageSettingsButtonPressed }
-						/>
-					</InspectorControls>
```

Comment:
`TollbarButton` which opens the BottomSheet is moved to `BlockMobileTolbar`. It will be visible only when there is some settings wrapped in `InspectorControls` block inside block